packages/react-components/src/components/TextField/TextField.css

```diff
@@ -122,3 +122,9 @@
   font: var(--typography-regular-small-body);
   color: var(--typography-color-danger);
 }
+
+.bcds-react-aria-TextField--container > .iconError {
```

Comment:
Let's change this to target the SVG directly to avoid the extra `<div>`:

```css
.bcds-react-aria-TextField--container > svg {
  color: var(--icons-color-danger);
  min-width: var(--icons-size-medium);
  height: var(--icons-size-medium);
}
```

```tsx
{isInvalid && <SvgExclamationIcon />}
```

Reply:
Oh wait nevermind — forgot we can target an SVG by its ID: 

```css
.bcds-react-aria-TextField[data-invalid]
  > .bcds-react-aria-TextField--container
  > **svg#exclamation-icon** {
  color: var(--icons-color-danger);
  min-width: var(--icons-size-medium);
  height: var(--icons-size-medium);
}
```

Gives us: 

<img width="509" alt="Screenshot 2024-09-11 at 13 53 08" src="https://github.com/user-attachments/assets/2df75535-d2f4-48ad-88cf-f4ec35165018">

Reply:
Fixed in [3321173](https://github.com/bcgov/design-system/pull/483/commits/33211732874c75c10450244fed1acb3caf087dce)